app/controllers/api/v2/courses_controller.rb

```diff
@@ -56,6 +56,14 @@ def update
         enrichment.assign_attributes(update_params)
         enrichment.save
 
+        site_ids = params[:course][:sites_ids]
+        if site_ids.present?
+          new_sites = site_ids
+            .map(&:to_i)
+            .map { |site_id| @provider.sites.find { |site| site.id == site_id } }
```

Comment:
```suggestion
            .map { |site_id| @provider.sites.find { |site| site.id == site_id.to_i } }
            
            
```

Thro should ` site_ids = params[:course][:sites_ids]` be a list of ints already?

Reply:
@defong They wouldn't be, these are params taken verbatim from the query string, I'm pretty sure they don't get converted.

Also, I prefer converting them as a separate step, as done in the pr, so that there's less to reason about on this line.